NDTensors/src/lib/UnallocatedArrays/src/abstractunallocatedarray.jl

```diff
@@ -0,0 +1,52 @@
+using FillArrays: FillArrays, getindex_value
+using NDTensors.SetParameters: set_parameters
+using Adapt: adapt
+
+const UnallocatedArray{ElT,N,AxesT,AllocT} = Union{
+  <:UnallocatedFill{ElT,N,AxesT,AllocT},<:UnallocatedZeros{ElT,N,AxesT,AllocT}
```

Comment:
I think you don't need `<:` here.

Reply:
I verified the `<:` is unnecessary and have removed them